### ISSUE TITLE
Forced uninstall of package and dependencies w/ meta package dependency throws error

### ISSUE THREAD

**Issue by dragon788** [chocolatey/choco#1074]
### What You Are Seeing?
Attempting to force uninstall a package and the dependency(ies) it pulled in fails if one of the dependencies is a meta package. This doesn't seem to fail if not forced.

### What is Expected?
Uninstalling a package with `--force-dependencies` removes the meta package and the packages it pulled in, AND the dependencies of the package it pulled in without exploding.

### How Did You Get This To Happen? (Steps to Reproduce)
Can be reproduced via this Gist.
https://gist.github.com/7a7524ddc2946b2b5d114ce615465697

### Output Log
I built the Gist on AppVeyor to make sure it wasn't just my local "Chocolatey Test Environment" VM, this is like a gazillion lines, so may take a bit to load.
https://ci.appveyor.com/project/dragon788/7a7524ddc2946b2b5d114ce615465697/build/1.0.2

The failure occurs right after this:
https://ci.appveyor.com/project/dragon788/7a7524ddc2946b2b5d114ce615465697/build/1.0.2#L337

This is the beginning of the debug output, "here be dragons".
https://ci.appveyor.com/project/dragon788/7a7524ddc2946b2b5d114ce615465697/build/1.0.2#L641

Here is the meat (what little there is) of the failure, this is the uninstall of the package and its dependencies.
https://ci.appveyor.com/project/dragon788/7a7524ddc2946b2b5d114ce615465697/build/1.0.2#L2878

**Comment by ferventcoder** [comment]
This certainly is something we'd want to fix.

**Comment by ferventcoder** [comment]
I'm guessing the relevant part is 

~~~sh
javaruntime v8.0.101 (forced)
 Skipping auto uninstaller - No registry snapshot.
elasticsearch not uninstalled. An error occurred during uninstall:
 Could not find a part of the path 'C:\ProgramData\chocolatey\lib\javaruntime\javaruntime.nupkg'
~~~

Which makes me think it is https://github.com/chocolatey/choco/issues/1040

**Comment by dragon788** [comment]
@ferventcoder Yeah, I looked at #1040 when you mentioned it on Gitter and as far as I can tell from the logs the casing isn't actually different here, but it's the same error message. Perhaps there is too strict of an equality comparison happening on the strings and it is blowing up in some way?

**Comment by ferventcoder** [comment]
That's kind of what I was thinking.

**Comment by pauby** [comment]
Thanks for raising this issue. After reviewing it, it's not something we expect to prioritise in the foreseeable future. We’re closing it to keep our backlog focused, but we’re open to revisiting it in the future if priorities shift or new interest arises. You are welcome to comment here or open a new issue if you'd like to discuss it further down the line.
